.github/workflows/locale-updater.yml

```
@@ -0,0 +1,54 @@
+      - develop
+  workflow_dispatch: {}
+
+jobs:
+  update-locales:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)